### ISSUE TITLE
Create Next App tests stall on Windows

### ISSUE THREAD

**Issue by ijjk** [vercel/next.js#11778]
When running `yarn` inside of `execa` while piping `stdin` `yarn` stalls causing the tests to stall on Windows. When using `inherit` on `stdin` instead `yarn` it does not stall. We need to investigate further before we can re-enable these tests on Windows

## System information

- OS: Windows 10
- Browser (if applies) [e.g. chrome, safari]: DNA
- Version of Next.js: [e.g. 6.0.2] latest
- Version of Node: 10.20.0

**Comment by Timer** [comment]
We removed the ability to choose an example interactively, so this is no longer relevant.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
